poweredby.zh.md

```diff
@@ -63,6 +63,10 @@ Apache Flink 为全球许多公司和企业的关键业务提供支持。在这
     <img src="{{ site.baseurl }}/img/poweredby/king-logo.png" alt="King" /><br />
       King，Candy Crush Saga的创建者，使用 Flink 为数据科学团队提供实时分析仪表板。<br><br><a href="https://techblog.king.com/rbea-scalable-real-time-analytics-king/" target='_blank'><small><span class="glyphicon glyphicon-new-window"></span></small> 阅读 King 的 Flink 实现</a>
   </div>
+  <div class="powered-by-tile col-md-3 col-sm-4 col-xs-6">
+    <img src="{{ site.baseurl }}/img/poweredby/klaviyo-logo.png" alt="Klaviyo" /><br />
+      Klaviyo使用 Apache Flink 扩展其实时分析系统，该系统每秒对超过一百万个事件进行重复数据删除和聚合。<br><br><a href="https://klaviyo.tech/tagged/counting" target='_blank'><small><span class="glyphicon glyphicon-new-window"></span></small> Read about real-time analytics at Klaviyo</a>
```

Comment:
"Read about real-time analytics at Klaviyo" should also be translated.

Reply:
@wuchong  thank you for your review, I will add it

Reply:
@wuchong I have updated it,pls check,thanks